### PR TITLE
Update CONTRIBUTING.md with additional requirements for doc

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
I'm going to make another small tweak, stand by.

**Comment by pshipton** [comment]
Updated.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,9 +53,18 @@ Following these guidelines will help us merge your pull requests smoothly:
 
 4. If your contribution introduces an external change that requires an update
    to the [user documentation](https://www.eclipse.org/openj9/docs/), add the
-   label `doc:externals` and open an  [issue](https://github.com/eclipse-openj9/openj9-docs/issues/new?template=new-documentation-change.md)
-   at the user documentation repository. Examples of an external change include
+   label `doc:externals` to the OpenJ9 Issue, or request a committer to do it,
+   and open an  [issue](https://github.com/eclipse-openj9/openj9-docs/issues/new?template=new-documentation-change.md)
+   at the user documentation repository. The OpenJ9 Pull Request should be labeled with `depends:doc` as well. Committers
+   should not merge the OpenJ9 Pull Request until at least a doc Issue is created, and ideally not until the doc Pull
+   Request is ready for merge. Rather than doing it separately, the Pull Request should add or update the
```
Comment (keithc-ca) [inline CONTRIBUTING.md:60]: I don't think 'Pull Request' needs to be capitalized; it should be lower-case like on line 66.
Same for 'Issue'.

Comment (pshipton) [reply CONTRIBUTING.md]: updated

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**